File: solutions/go/rna-transcription/1/rna_transcription.go
package strand

import (
	"strings"
)

var strands = map[string]string{
	"C": "G",
	"G": "C",
	"T": "A",
	"A": "U",
}

func ToRNA(dna string) string {
	var r string
	s := strings.Split(dna, "")
	if len(s) == 0 {
		return ""
	}
	for _, v := range s {
		r = r + strands[v]
	}
	return r
}

File: solutions/javascript/rna-transcription/1/rna-transcription.js
const DNA = {
  G: "C",
  C: "G",
  T: "A",
  A: "U",
};
export const toRna = (input) => {
  return input
    .split("")
    .map((nucleotide) => DNA[nucleotide])
    .join("");
};
